modular_zubbers/code/datums/votes/gamemode_vote.dm

```diff
@@ -0,0 +1,30 @@
+// BUTTERSTATION ADDITION
+
+#define CHOICE_EXTENDED "Extended Ruleset (Minimal Threats)"
```

Comment:
```suggestion
#define CHOICE_EXTENDED "Extended, potentially with OPFORs"
```

Reply:
My line of thought was there's always gonna be some amount of threat, whether that's admeme intervention, self antaggers or opfors, so I feel like minimal is the best way to put it.